rllib/agents/pg/pg.py

```diff
@@ -1,9 +1,32 @@
+"""
```

Comment:
Add comment header to each main algo file, citing the papers.

Reply:
Looks great.